Python/getargs.c

```diff
@@ -859,6 +859,15 @@ convertsimple(PyObject *arg, const char **p_format, va_list *p_va, int flags,
         break;
     }
 
+    case 'P': {/* boolean predicate with _Bool */
+        bool *p = va_arg(*p_va, bool *);
+        int val = PyObject_IsTrue(arg);
+        if (val < 0)
+            RETURN_ERR_OCCURRED;
+        *p = (bool)(val);
```

Comment:
```suggestion
        *p = (bool)val;
```

or 

```suggestion
        *p = (bool) val;
```

Depending on the style of the file.